yang/frr-vrf.yang

```diff
@@ -0,0 +1,61 @@
+module frr-vrf {
+  yang-version 1.1;
+  namespace "http://frrouting.org/yang/vrf";
+  prefix frr-vrf;
+
+  organization
+    "Free Range Routing";
+  contact
+    "FRR Users List:       <mailto:frog@lists.frrouting.org>
+     FRR Development List: <mailto:dev@lists.frrouting.org>";
+  description
+    "This module defines a model for managing FRR VRF.";
+
+  revision 2019-12-06 {
+    description
+      "Initial revision.";
+  }
+
+  feature netns;
+
+  container lib {
+    list vrf {
+      key "name";
+      description
+        "VRF.";
+      leaf name {
+        type string {
+          length "1..16";
+        }
+        description
+          "VRF name.";
+      }
+
+      leaf id {
+        type uint32 {
+          range "1..4294967295";
+        }
```

Comment:
I think this should be `config false` since we can't configure VRF IDs manually as far as I know.

Talking about operational data, it would be good to have an `active` leaf as well indicating whether the VRF is active in the kernel.

Reply:
Added `config false` to `id` member leaf. Added new `active` leaf to indiacate vrf is active in kernel.